src/libs/API/types.ts

```diff
@@ -187,6 +187,7 @@ const WRITE_COMMANDS = {
     DELETE_POLICY_TAXES: 'DeletePolicyTaxes',
     UPDATE_POLICY_TAX_VALUE: 'UpdatePolicyTaxValue',
     RENAME_POLICY_TAX: 'RenamePolicyTax',
+    UPDATE_CONNECTION_CONFIG: 'UpdateConnectionConfig',
```

Comment:
```suggestion
    UPDATE_POLICY_CONNECTION_CONFIGURATION: 'UpdatePolicyConnectionConfiguration',
```

The name of the command in web-E is 'UpdatePolicyConnectionConfiguration'

Reply:
fixed https://github.com/Expensify/App/pull/38006/commits/bb7090d19a2487c0746961acf84f1691028407ee 🙇